src/Chainweb/Pact/Backend/RelationalCheckpointer.hs

```diff
@@ -277,8 +280,7 @@ doGetEarliestBlock db = do
     [] -> return Nothing
     (!o:_) -> return (Just o)
   where
-    qtext = "SELECT blockheight, hash FROM BlockHistory \
-            \ ORDER BY blockheight ASC LIMIT 1"
+    qtext = "SELECT blockheight, hash FROM BlockHistory ORDER BY blockheight ASC LIMIT 1"
```

Comment:
String literals with gaps break CPP. It's probably generally a good idea to avoid them and wait for new multi-line string literals to become available in GHC (https://github.com/ghc-proposals/ghc-proposals/blob/master/proposals/0569-multiline-strings.rst).